lib/engine/game/g_18_uruguay/step/dividend.rb

```diff
@@ -39,7 +39,10 @@ def description
           end
 
           def auto_actions(entity)
-            return [Engine::Action::Dividend.new(current_entity, kind: 'withhold')] if entity.loans.size.positive?
+            if entity.loans.size.positive? && @game.nationalized?
+              return [Engine::Action::Dividend.new(current_entity,
+                                                   kind: 'withhold')]
+            end
 
             []
```

Comment:
Testing `Array.size.positive?` isn't idiomatic Ruby – the `Array.empty?` method is the way to do this test. I'd also suggest reordering this method, I think it looks clearer like this:
```suggestion
            return [] unless @game.nationalized?
            return [] if entity.loans.empty?

            [Engine::Action::Dividend.new(current_entity,kind: 'withhold')]
```

Reply:
Agree it looks cleaner, updated according to suggestion